Kernel/FileSystem/InodeWatcher.cpp

```diff
@@ -101,8 +101,10 @@ void InodeWatcher::notify_inode_event(Badge<Inode>, InodeIdentifier inode_id, In
 
         m_queue.with([watcher, event_type, name](auto& queue) {
             OwnPtr<KString> path;
-            if (!name.is_null())
-                path = KString::try_create(name).release_value_but_fixme_should_propagate_errors();
+            if (!name.is_null()) {
+                if (auto result = KString::try_create(name); !result.is_error())
```

Comment:
I don't think this is correct. Emitting an add_child inode watcher event without a path should be wrong. `name` can only be null for certain events where we don't need a path.